go.mod

```diff
@@ -3,14 +3,18 @@ module github.com/pokt-network/poktroll
 go 1.22.2
 
 replace (
+	// fix upstream GHSA-h395-qcrw-5vmq vulnerability.
```

Comment:
:mechanical_arm:

Reply:
Actually - good call out, we don't need this anymore. I've removed this replace and gin was bumped to 1.9.0.